custom_components/sat/climate.py

```
@@ -36,6 +35,7 @@
 from homeassistant.helpers.event import async_track_state_change_event, async_track_time_interval
 from homeassistant.helpers.restore_state import RestoreEntity
 
+from .area import Areas, SENSOR_TEMPERATURE_ID
 from .boiler_state import BoilerState
 from .const import *
```

Ruff (0.8.2)

`from .const import *` used; unable to detect undefined names

(F403)